lib/engine/corporation.rb

```diff
@@ -99,6 +100,11 @@ def corporation?
       true
     end
 
+    def push_revenue_history!(revenue, round_num)
```

Comment:
how about just add_revenue

Reply:
2 hard problems in computer science. will do